README.md

```diff
@@ -191,3 +191,11 @@ def build_new() -> DataFrame:
    
 ```
    
+### Adding Certificates
+
+To allow developers to include their required CA certificates from their local machine, the below instructions need to be added to the Dockerfile
+
+```dockerfile
+   COPY certs/ /usr/local/share/ca-certificates/
+   RUN update-ca-certificates
```

Comment:
Don't forget to add this two command (COPY, RUN) to the actual Dockerfile. That way users behind proxy will only need to add certificate and will not need modify dockerfile itself